lib/cylc/review_dao.py

```diff
@@ -218,8 +221,12 @@ def get_suite_job_entries(
                 return ([], 0)
         except sqlite3.Error:
             return ([], 0)
-
-        if self.is_cylc8(user_name, suite_name):
+        from cylc.review import CylcReviewService
```

Comment:
This is imported down here due to a circular import problem. Unsure if there is a better way of solving this.

Reply:
Possible alternative - make `is_cylc8` a module function (not a class method) and using
`from cylc.review import is_cylc8`. Might look more elegant.

Reply:
This import is also needed for the call to `_get_user_home`, not just for `is_cylc8` so making it a module function won't help with that unfortunately. I did look at moving this out but I don't think a big refactor is a good idea.